helm/primehub/scripts/install-tools.sh

```diff
@@ -0,0 +1,37 @@
+mkdir /data
+cd /data
+
+wget https://storage.googleapis.com/kubernetes-release/release/v1.14.1/bin/linux/amd64/kubectl -O kubectl
```

Comment:
I change the image to **gcr.io/google-containers/hyperkube-amd64:v1.14.0** ,because `/kubectl` in hyperkube images is a symbolic link to `/hyperkube` and hyperkube has native library dependecies not able to run in a busybox cotainer by itself.